src/data/team.js

```diff
@@ -266,6 +266,21 @@ Various design podcasts including [SimpleBits](https://show.simplebits.com/), [H
     gif: 'https://user-images.githubusercontent.com/6720549/118991044-63a24300-b951-11eb-81d4-d5a63f71db06.gif',
     alumni: false,
   },
+  {
```

Comment:
Folks typically add themselves at the top of this file so newer team members will show up first in the listing on primer.style/about. I'd recommend moving this up to the top of the file so you'll show up first!